src/main/webapp/components/SingleInputComponent.jsx

```diff
@@ -156,7 +156,7 @@ function SingleInputComponent(props) {
                 {options && options.length > 0 && options}
             </SelectWrapper>
             {effectiveIsClearable ? (
-                <StyledButton
```

Comment:
Why do we revert to use just a Button instead of StyledButton?

Reply:
We used StyledButton so that any button in the application (like Cancel, Add, etc.) should be visible in the same width, but what is here is the option to clear the value of the field that is showing beside the dropdown (as we had changed globally, it is also applicable here), so I am reverting to normal mode.

Before:
<img width="707" alt="Screenshot 2023-02-24 at 2 54 03 PM" src="https://user-images.githubusercontent.com/62089106/221142666-eb794fe3-77f2-46a9-9ce7-6000162084ca.png">

After:
<img width="594" alt="Screenshot 2023-02-24 at 2 55 13 PM" src="https://user-images.githubusercontent.com/62089106/221142711-54953810-2e37-4b08-b5df-a71a13a6e0dc.png">